packages/component-library/src/components/form/mt-text-editor/_internal/mt-text-editor-toolbar-button-link.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <mt-text-editor-toolbar-button
+    :button="button"
+    :editor="props.editor"
+    :disabled="disabled"
+    @click="openLinkModal"
+  />
+
+  <!-- Link modal -->
+  <mt-modal-root :isOpen="showLinkModal" @change="($event) => (showLinkModal = $event)">
+    <mt-modal width="s" :title="t('mt-text-editor-toolbar-button-link.modalTitle')">
+      <template #default>
+        <div class="mt-text-editor__link-modal">
+          <mt-text-field
+            :label="t('mt-text-editor-toolbar-button-link.linkUrl')"
+            v-model="linkHref"
+            placeholder="https://example.com"
+            required
+          />
+          <mt-switch
+            :label="t('mt-text-editor-toolbar-button-link.openInNewTab')"
+            :checked="linkTarget === '_blank'"
+            @change="
+              (checked) => {
+                linkTarget = checked ? '_blank' : '';
+              }
+            "
+            :aria-label="t('mt-text-editor-toolbar-button-link.openInNewTab')"
+          />
+        </div>
+      </template>
+      <template #footer>
+        <div class="mt-text-editor__link-modal-footer">
+          <mt-button variant="secondary" @click="() => (showLinkModal = false)">
```

Comment:
You can use the `mt-modal-close` component, just like you would use the `mt-modal-trigger` component to close the modal

Reply:
Same as above: I will change it. Documentation for this would be also cool :)